easybuild/easyconfigs/s/Sphinx/Sphinx-1.4.6-foss-2016a-Python-2.7.11.eb

```diff
@@ -0,0 +1,77 @@
+easyblock = 'Bundle'
+
+name = 'Sphinx'
+version = '1.4.6'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://sphinx.pocoo.org/'
+description = """Sphinx is a tool that makes it easy to create intelligent and beautiful documentation.
+ It was originally created for the new Python documentation, and it has excellent facilities for the 
+ documentation of Python projects, but C/C++ is already supported as well, and it is planned to add 
+ special support for other languages as well."""
+
+toolchain = {'name': 'foss', 'version': '2016a'}
+
+# this is a bundle of Python packages
+exts_defaultclass = 'PythonPackage'
+
+dependencies = [
+    ('Python', '2.7.11'),
+    ('Pygments', '2.1.3', '-Python-%(pyver)s'),
+    ('requests', '2.10.0', versionsuffix),
+]
+
+patches = [
+    '%(name)s-%(version)s-pdflatex_failing_tests.patch'
```

Comment:
@RvDijk the patch will not be applied this way, you need to specify it in the `exst_list` entry for Sphinx itself below:

```python
(name, version, {
    'source_urls': ['https://pypi.python.org/packages/source/S/Sphinx/'],
    'patches': ['Sphinx-%(version)s-pdflatex_failing_tests.patch'],
}),
```

(I'm not sure `%(version)s` will work there, you'll need to check that)